README.md

```diff
@@ -9,7 +9,7 @@ VCMI is work-in-progress attempt to recreate engine for Heroes III, giving it ne
  * Homepage:   https://vcmi.eu/
  * Wiki:       https://wiki.vcmi.eu/
  * Forums:     https://forum.vcmi.eu/
- * Bugtracker: https://bugs.vcmi.eu/
+ * Bugtracker: https://github.com/vcmi/vcmi/issues

```

Comment:
There is incorrect link to bugtracker fromm the website too, would be good to fix it too https://github.com/vcmi/VCMI.eu

Reply:
Good point. I'll open a separate PR for that